flixel/FlxGame.hx

```diff
@@ -758,8 +759,6 @@ class FlxGame extends Sprite
 		}
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
-		
-		filters = filtersEnabled ? _filters : null;
```

Comment:
Yes... it is arguable whether it's worth making this change. Maybe someone else can weigh in on it?